SDPi_Supplement/asciidoc/volume2/gateways/tf2-ch-b-gateway-obx18-mapping.adoc

```diff
@@ -0,0 +1,71 @@
+The OBX-18 field shall contain the equipment (or device) identifier on the MDS level and/or the measurement module identifier of the equipment on the VMD level.

```

Comment:
See https://github.com/IHE/sdpi-fhir/pull/163/files#r1012884811

Reply:
Maybe it is just me because I am an HL7 N00b, but did we reach the point at which we should come up with a definition of _equipement label_ / _device name_ / _soft id_ and _measurement module identifier_? How is the _measurement module identifier_ modelled in BICEPS?

Reply:
Hopefully, we can discuss this in the week Nov. 28th when we meet in Luebeck again.